src/store/store.jsx

```diff
@@ -1,8 +1,10 @@
 import { configureStore } from '@reduxjs/toolkit';
 import productSlice from '../features/todo/todoSlice';
+import allProductSlice from '../features/all_products/all_products';
 
 export const store = configureStore({
     reducer: {
-        product: productSlice
+        product: productSlice,
+        productList: allProductSlice,
```

Comment:
I'm really confused what happens in this file.
1. you import `productSlice` from `todo/todoSlice`
2. you import `allProductSlice` from `all_products` and then use it as a value for `productList` property
naming is very confusing in this file